### ISSUE TITLE
Make an ISO with two SKUs of windows

### ISSUE THREAD

**Issue by MrArdillo** [ChrisTitusTech/winutil#1995]
Can you make a ISO with two or more versions of Windows? Like for example a ISO with professional and home or professional and education or the three in one.

**Comment by KonTy** [comment]
@MrArdillo  not currently, the thinking behind MicroWin is to make an unattend image there you don't have to enter anything.

**Comment by MrArdillo** [comment]
Okay, but can we have the possibility in the future to make and image with more skus to save storage not having two images from two Skus?

**Comment by CodingWonders** [comment]
@KonTy, I saw this recently and thought it would be interesting, so I made an activity diagram to plan how this could be done.

[MicroWin_AppendImage.pdf](https://github.com/user-attachments/files/15977029/MicroWin_AppendImage.pdf)

We would rely on the `Append-Image` flag of DISM, as stated in the PDF file. Given the nature of WIM files, where the images are compressed with LZX compression by default, the target ISO file would not be gigantic.
